.gitignore

```diff
@@ -8,8 +8,5 @@ test/dummy/.sass-cache
 .sass-cache/
 /lit-*.gem
 **.DS_Store
-Gemfile.lock
```

Comment:
Let's have a `Gemfile` checked into the repository, just like this:
```
# frozen_string_literal: true

source 'https://rubygems.org'

gemspec
```
This way it'll be in place for gems like Appraisal that like it being in place, but use the gemspec to retrieve dependencies.

Also note that Yehuda Katz [does not recommend checking in Gemfile.lock into gem repositories](https://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/) so we should probably keep it in `.gitignore`. I'm fine with having lockfiles for each appraisal though.